eoscostarica/eos-rate
# Secure websockets

### GH Issue

insecure WebSocket endpoint #261

### Steps to test

1. visit eosrate.io
2. open JS console
3. no SSL warning should appear

### Checklist
